docs/quick_tutorial/authorization/setup.py

```diff
@@ -7,8 +7,15 @@
     'waitress',
 ]
 
+tests_require = [
+    'pyramid[testing]'
```

Comment:
no this needs to list out the dependencies you want to install... such as webtest. Please remove `pyramid[testing]`.

Reply:
also as I said in the previous comment you need to update the instructions above to say `pip install -e ".[testing]"', not just `pip install -e .` so that these dependencies get installed.

Reply:
This is a common gotcha with the Quick Tutorial: most steps are based on previous steps' code and manually installed dependencies, yet we don't consistently update the `setup.py`'s dependencies to match the manual installation.

Tonight I will audit when dependencies are added manually against added to `setup.py` to ensure that confusion does not happen. I might also insert a step about how to manage dependencies manually without tools (because controversy is fun).

Reply:
I've completed my audit in PR #3379 and merged it into `master`.

@tomascorrea would you please compare these files against `master`, and either rebase or update, then push your changes to this PR?